dev/07_vision_core.ipynb

```diff
@@ -500,7 +527,7 @@
    "outputs": [],
    "source": [
     "camvid = untar_data(URLs.CAMVID_TINY)\n",
-    "fns = get_image_files(camvid)\n",
+    "fns = get_image_files(camvid, folders='images')\n",
```

Comment:
Else get_image_files() grabbed masks with _P suffix also; and files with *_P_P.png were not found.